MigratoryBirds/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+using System.Linq;
+using System.Collections.Generic;
+
+public class Program
+{
+    public static void Main(string []args){
+        var a = migratoryBirds(new List<int>{1,4,4,4,5,3});
+    }
+
+    public static int migratoryBirds(List<int> arr)
+    {

```

Comment:
why this function is return 0.

Reply:
Resolved, now it's returning expected value.